README.md

```diff
@@ -1,2 +1,35 @@
 # layer5-repo-template
 This repository is used as the boilerplate for consistency across all Layer5 repos.
+
+div>&nbsp;</div>
+
+## Join the service mesh community!
+
+<a name="contributing"></a><a name="community"></a>
+Our projects are community-built and welcome collaboration. 👍 Be sure to see the <a href="https://docs.google.com/document/d/17OPtDE_rdnPQxmk2Kauhm3GwXF1R5dZ3Cj8qZLKdo5E/edit">Layer5 Community Welcome Guide</a> for a tour of resources available to you and jump into our <a href="http://slack.layer5.io">Slack</a>!
+
+<p style="clear:both;">
+<a href ="https://layer5.io/community"><img alt="MeshMates" src="docs/assets/images/Layer5-MeshMentors.png" style="margin-right:10px; margin-bottom:7px;" width="28%" align="left" /></a>
+<h3>Find your MeshMate</h3>
+
+<p>MeshMates are experienced Layer5 community members, who will help you learn your way around, discover live projects and expand your community network. 
+Become a <b>Meshtee</b> today!</p>
+
+Find out more on the <a href="https://layer5.io/community#meshmate">Layer5 community</a>. <br />
+<br /><br /><br /><br />
+</p>
+
+<a href="https://meshery.io/community"><img alt="Layer5 Service Mesh Community" src="docs/assets/img/readme/slack-128.png" style="margin-left:10px;padding-top:5px;" width="110px" align="right" /></a>
+
+<a href="http://slack.layer5.io"><img alt="Layer5 Service Mesh Community" src="docs/assets/img/readme/community.png" style="margin-right:8px;padding-top:5px;" width="140px" align="left" /></a>
+
+<p>
+✔️ <em><strong>Join</strong></em> any or all of the weekly meetings on <a href="https://calendar.google.com/calendar/b/1?cid=bGF5ZXI1LmlvX2VoMmFhOWRwZjFnNDBlbHZvYzc2MmpucGhzQGdyb3VwLmNhbGVuZGFyLmdvb2dsZS5jb20">community calendar</a>.<br />
+✔️ <em><strong>Watch</strong></em> community <a href="https://www.youtube.com/playlist?list=PL3A-A6hPO2IMPPqVjuzgqNU5xwnFFn3n0">meeting recordings</a>.<br />
+✔️ <em><strong>Access</strong></em> the <a href="https://drive.google.com/drive/u/4/folders/0ABH8aabN4WAKUk9PVA">community drive</a>.<br />
+</p>
+<p align="center">
+<i>Not sure where to start?</i> Grab an open issue with the <a href="https://github.com/issues?utf8=✓&q=is%3Aopen+is%3Aissue+archived%3Afalse+org%3Alayer5io+label%3A%22help+wanted%22+">help-wanted label</a>.
+</p>
+
+<div>&nbsp;</div>
```

Comment:
Perfect.

Reply:
@leecalcote I didn't know someone was already working on this, as there was no issue or any other note that a task was assigned.   I just saw the template and thought I'd pitch in, so I did this last night. Looks like someone else did the same tasks this morning and merged theirs first, so there were conflicts between the files.  I've certainly learned my lesson!

Reply:
@morrme no, no. Your PR is great and your intuition is spot on. 👍 

The conversations held in the community meetings, the Slack workspace, and the community Google Docs (you're welcome to any of those), don't always make their way into GitHub. There's much context missing here, which naturally leads to some amount of confusion. No sweat though.

Reply:
@leecalcote I'd decided to leave this alone, but for the good of anyone else who comes after me I will say this : I have tried to be supportive and helpful, and even spread the word and welcomed/helped others to contribute. However, over and over I've felt very less than welcome myself and it has made me regret my involvement.  The first time this happened, I actually did step away and decline joining, because I did not want to be aligned with people who see open source contributions as a competitive sport. I thought maybe I had judged too soon, so I cautiously began helping with this and other issues again the other day. In this case, I don't believe there was any confusion, especially with the person involved. Essentially, whether they were assigned a task "offline" or not, they submitted the same work that I did, 8 hours after I did, merged their own changes without review , *then* "reviewed" mine and even asked that I fix a conflict that I can only guess occurred due to their actions. Instead of their behavior being corrected, It was then inferred that it was my fault because I chose to not join "offline". That's unfair and frustrating, and it completely voids all of the "community" themed effort. If you do indeed desire to foster a community, you must seek the discernment to ensure those that you allow to represent your organization share the same values. I wish you the best in that quest.